website/docs/topics/concepts/what-is-a-feature-flag.md

```diff
@@ -1,135 +1,211 @@
 ---
-title: What is a feature flag?
+title: What is a feature flag and why are feature flags used?
 slug: /what-is-a-feature-flag
-description: Feature flags, also known as feature toggles, allow developers to enable or disable features or code paths dynamically.
+description: Feature flags let you control software features in real time, enabling safer deployments, better testing, and faster innovation.
 ---
 
-Feature flags, also known as feature toggles, allow developers to enable or disable features or code paths dynamically. This approach helps manage the deployment of new features, perform A/B testing, roll out features gradually, and provide quick rollback capabilities.
+import Figure from '@site/src/components/Figure/Figure.tsx'
 
-### The Concept
+Feature flags allow you to release, test, and manage features and functionality across your application without changing the source code. Organizations use added control and flexibility to deliver more and higher quality features with reduced cost, time, and risk.
 
-Feature flags work by wrapping a piece of code with a conditional statement that checks the status of a variable, called a flag. If the flag is enabled, the new code executes; if not, the existing or alternative code path runs. That flag can then be toggled (hence, feature toggles) dynamically at runtime. This enables features to be merged into the main codebase but remain dormant until they are ready to be activated.
+In this guide, we’ll introduce you to the basics of feature flags and why so many development teams choose them.
 
-<Figure img="/img/feature-flag-example.png" caption="At its most basic level, a feature flag is a conditional statement that checks the status of a variable. The value of the variable determines if the code path is enabled or disabled."/>
+## Why developers use feature flags
 
+In 2014, [Ivar Østhus](https://www.linkedin.com/in/ivarconr/overlay/about-this-profile/), our CTO, was working as a software developer at FINN.no, Norway’s largest online marketplace. Getting new features into production was a pain, so, as a developer, he looked around and wondered if he could write some code to make it easier. His initial goal was to protect features under development, allowing his team to transition to [trunk-based development](https://www.getunleash.io/blog/how-to-implement-trunk-based-development-a-practical-guide)—a practice recommended by the [State of DevOps Report](https://cloud.google.com/devops/state-of-devops).
```

Comment:
I like that idea and agree that 1st person is more impactful here.  Does it need to be collapsible?  I think we can just switch back to 1st person, no?

Reply:
I think it would be a good idea to go for something different visually than the rest of the text. Just to help the reader switch between the different narratives. I went for this option:

https://unleash-docs-git-melinda-what-is-a-feature-flag-unleash-team.vercel.app/what-is-a-feature-flag#why-developers-use-feature-flags

wdyt?